modules/post/windows/gather/enum_application_artifacts_packrat.rb

```diff
@@ -0,0 +1,918 @@
+##
+# This module requires Metasploit: http//metasploit.com/download
```

Comment:
$ tools/msftidy.rb modules/post/windows/gather/enum_application_artifacts_packrat.rb
modules/post/windows/gather/enum_application_artifacts_packrat.rb - [INFO] Invalid URL: # This module requires Metasploit: http//metasploit.com